lib/presentation/pages/top/top_page.dart

```diff
@@ -2,66 +2,62 @@ import 'package:count_habits/presentation/pages/settings/settings_page.dart';
 import 'package:count_habits/presentation/pages/top/components/middle_navigation_items.dart';
 import 'package:count_habits/presentation/pages/top/components/middle_navigaton_bar.dart';
 import 'package:count_habits/presentation/pages/top/components/summary_card.dart';
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
```

Comment:
<table data-meta="generated_by_danger-pr">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kolbe-ryo/count_habits/blob/e39470e55f7fefda91212c1332f83faf02699b1a/lib/presentation/pages/top/top_page.dart#L6"></span>The import of 'package:flutter/material.dart' is unnecessary because all of the used elements are also provided by the import of 'package:flutter/cupertino.dart'</td>
    </tr>
  </tbody>
</table>